backend/tests/DataModeling.Tests/DataModeling.Tests.csproj

```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Altinn.Studio.DataModeling" />
+    <PackageReference Include="Altinn.Studio.DataModeling" Version="1.0.0"/>
```

Comment:
I think this package reference can be removed since we're referencing the project.